src/main/java/org/apache/sling/jcr/repoinit/impl/AclVisitor.java

```diff
@@ -142,7 +142,9 @@ public void visitCreatePath(CreatePath cp) {
         for (PathSegmentDefinition psd : cp.getDefinitions()) {
             final String fullPath = parentPath + "/" + psd.getSegment();
             try {
-                if (session.itemExists(fullPath)) {
+                if (session.propertyExists(fullPath)) {
+                    throw new RuntimeException("Error creating Node at " + fullPath + ": a property exists with the same path");
+                } else if (session.itemExists(fullPath)) {
```

Comment:
i would rather use 'session.nodeExists(fullPath)' here just for the sake of clarity and also to make really sure we don't by mistake have a property introduced just now somehow.... not very likely but i have seen all sort of unlikely nonsense.